src/main/scala/nl.knaw.dans.easy.deposit/Command.scala

```diff
@@ -27,22 +28,25 @@ object Command extends App with DebugEnhancedLogging {
   type FeedBackMessage = String
 
   val configuration = Configuration(File(System.getProperty("app.home")))
-  val commandLine: CommandLineOptions = new CommandLineOptions(args, configuration) {
-    verify()
-  }
+  val commandLine: CommandLineOptions = new CommandLineOptions(args, configuration)
   Try { new EasyDepositApiApp(configuration) }
     .flatMap(runSubcommand)
     .doIfSuccess(msg => Console.err.println(s"OK: $msg"))
     .doIfFailure { case e => logger.error(e.getMessage, e) }
     .doIfFailure { case NonFatal(e) => Console.err.println(s"FAILED: ${ e.getMessage }") }
 
-  private def runSubcommand(app: EasyDepositApiApp): Try[FeedBackMessage] = {
+  def runSubcommand(app: EasyDepositApiApp) =
```

Comment:
Please keep the change as minimal as possible!

```suggestion
  private def runSubcommand(app: EasyDepositApiApp): Try[FeedBackMessage] = {
```